eng/dcppack/Aspire.Hosting.Orchestration.targets

```diff
@@ -1,9 +1,12 @@
 <Project>
 
   <PropertyGroup>
-    <DcpDir Condition=" '$(DcpDir)' == '' ">$([MSBuild]::NormalizePath($(MSBuildThisFileDirectory), '..', 'tools'))</DcpDir>
-    <DcpExtensionsPath Condition=" '$(DcpExtensionsPath)' == '' ">$([MSBuild]::NormalizePath($(DcpDir), 'ext'))</DcpExtensionsPath>
-    <DcpBinPath Condition=" '$(DcpBinPath)' == '' ">$([MSBuild]::NormalizePath($(DcpExtensionsPath), 'bin'))</DcpBinPath>
+    <DcpDir Condition=" '$(DcpDir)' == '' ">$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', 'tools'))</DcpDir>
+    <DcpDir>$([MSBuild]::EnsureTrailingSlash('$(DcpDir)'))</DcpDir>
+    <DcpExtensionsPath Condition=" '$(DcpExtensionsPath)' == '' ">$([MSBuild]::NormalizeDirectory($(DcpDir), 'ext'))</DcpExtensionsPath>
+    <DcpExtensionsPath>$([MSBuild]::EnsureTrailingSlash('$(DcpExtensionsPath)'))</DcpExtensionsPath>
+    <DcpBinPath Condition=" '$(DcpBinPath)' == '' ">$([MSBuild]::NormalizeDirectory($(DcpExtensionsPath), 'bin'))</DcpBinPath>
```

Comment:
nit: should the convention be for properties with names ending in `*Dir` to be directories, and `*Path` to be file paths? That seems to be the case in some other repos. Either way, having it consistent would mean that incorrect use/expectation of dir/file would be easy to spot when reading targets files.

Reply:
I'll look at standardizing this in a separate PR, since it would technically be a breaking change